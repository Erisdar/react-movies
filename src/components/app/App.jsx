import 'bootstrap/dist/css/bootstrap.min.css';
import React from 'react';
import './app.css';
import Content from './content/ContentContainer';
import Footer from './footer/Footer';
import Header from './header/Header';

export default () => (
    <>
        <Header />
        <Content />
        <Footer />
    </>
);