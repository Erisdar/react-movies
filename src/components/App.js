import 'bootstrap/dist/css/bootstrap.min.css';
import React from 'react';
import ReactDom from 'react-dom';
import '../styles/app.css';
import Content from './Content';
import Footer from './Footer';
import Header from './Header';

const App = () => (
    <>
        <Header />
        <Content />
        <Footer />
    </>
);

ReactDom.render(<App />, document.getElementsByClassName("app")[0]);
