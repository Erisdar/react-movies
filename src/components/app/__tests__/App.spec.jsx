import { shallow } from 'enzyme';
import React from 'react';
import App from '../App';

describe('AppContainer component', () => {
    it('should be render correctly', () => {
        const component = shallow(<App />);
        expect(component).toMatchSnapshot();
    });
});