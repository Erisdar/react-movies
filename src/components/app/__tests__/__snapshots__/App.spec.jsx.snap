// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer component should be render correctly 1`] = `
<Fragment>
  <_default />
  <ConnectFunction />
  <_default />
</Fragment>
`;
